src/js/tracks/html-track-element-list.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file html-track-element.js
+ */
+
+import * as browser from '../utils/browser.js';
+import document from 'global/document';
+
+class HtmlTrackElementList {
+  constructor(trackElements = []) {
+    let list = this;
+
+    if (browser.IS_IE8) {
+      list = document.createElement('custom');
+
+      for (let prop in HtmlTrackElementList.prototype) {
+        list[prop] = HtmlTrackElementList.prototype[prop];
```

Comment:
Just remembered, we should also make sure that we don't copy the `constructor` property because IE8 will complain.